groundwater/model-test.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.gis.db import models
+
+
+class Quantity(models.Model):
+    """Model to define quantity."""
+
+    unit = models.TextField(null=True, blank=True)
+    value = models.FloatField(null=True, blank=True)
+
+    class Meta:
+        managed = False
+        db_table = 'groundwater\".\"quantity'
+
+
+class Borehole(models.Model):
+    """Model to define groundwater borehole."""
+
+    id = models.AutoField(primary_key=True)
+    bholeDateOfDrilling = models.DateField(null=True)
+    bholeNominalDiameter = models.ForeignKey(
+        Quantity, null=True, blank=True, on_delete=models.CASCADE)
+
+    class Meta:
+        managed = False
+        db_table = 'groundwater\".\"borehole'
```

Comment:
I. think it will be cleaner to use django routing to deal with the separate schema. See here for example:

https://stackoverflow.com/a/51007441/3849375

Will you make a ticket to take care of that after our demo tomorrow?

Reply:
@timlinux this file is actually not used, it was accidentally commited